### PR TITLE
feat: support wechat pay

### PR DESCRIPTION
**Pull request by Chinoholo0807**
Fix: https://github.com/casdoor/casdoor/issues/2259

### CODE REVIEW

**Comment by Chinoholo0807** [comment]
https://github.com/casdoor/casdoor/assets/54203997/ffbdfbba-6020-4245-a48a-71679770b1da

File: web/src/QrCodePage.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+import React from "react";
+import {useEffect, useState} from "react";
+import QRCode from "qrcode.react";
+import {Col, Row} from "antd";
+import * as PaymentBackend from "./backend/PaymentBackend";
+import * as Setting from "./Setting";
+
+export default function QrCodePage({providerDisplay, owner, paymentName, payUrl, successUrl, size}) {
+  window.console.log(payUrl, successUrl);
```
Comment (hsluoyz) [inline web/src/QrCodePage.js:23]: Remove window.console.log

File: controllers/payment.go [hunk 2]
```diff
@@ -176,11 +176,10 @@ func (c *ApiController) DeletePayment() {
 func (c *ApiController) NotifyPayment() {
 	owner := c.Ctx.Input.Param(":owner")
 	paymentName := c.Ctx.Input.Param(":payment")
-	orderId := c.Ctx.Input.Param("order")
```
Comment (hsluoyz) [inline controllers/payment.go:179]: `orderId` is added in PR: https://github.com/casdoor/casdoor/pull/1959 , it's for PayPal, is it safe to remove?

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Comment by Chinoholo0807** [comment]
https://github.com/casdoor/casdoor/assets/54203997/6285f17c-bb58-41c3-bc47-3e3f1c38ff49

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**